h2o-py/h2o/backend/server.py

```diff
@@ -344,7 +345,10 @@ def _launch_server(self, port, baseport, mmax, mmin, ea, nthreads, jvm_custom_ar
         giveup_time = time.time() + self._TIME_TO_START
         while True:
             if proc.poll() is not None:
-                raise H2OServerError("Server process terminated with error code %d" % proc.returncode)
+                if proc.returncode == self._BAD_JAVA_VERSION_RETURN_CODE_:
```

Comment:
we should do the same for R (can be a separate PR)

Reply:
found this one where it is done: c86af86

Reply:
I think we should keep Python and R consisent and implement the same solution here

Reply:
stderr added to the the server error in a3da706